addons/l10n_gr/data/template/account.tax.group-gr.csv

```diff
@@ -1,4 +1,8 @@
-"id","name","country_id"
-"tax_group_19","ΦΠΑ 19%","base.gr"
-"tax_group_21","ΦΠΑ 21%","base.gr"
-"tax_group_23","ΦΠΑ 23%","base.gr"
+"id","name","country_id","tax_receivable_account_id","tax_payable_account_id","name@gr"
```

Comment:
It's really good that we have the tax_receivable_account_id and tax_payable_account_id, but we should use an account that is not already used on taxes.

(The account will be the counterpart account for the tax closing entry.)

I suggest using account l10n_gr_54_02_03 for both the tax_receivable_account_id and the tax_payable_account_id.

Reply:
if all are the same, can not we just set it on the chart template?